skins/laika/src/store/util.ts

```diff
@@ -7,3 +9,18 @@ function buildActionOrMutationName( ...namespacesAndName: string[] ): string {
 
 export const action = buildActionOrMutationName;
 export const mutation = buildActionOrMutationName;
+
+export const Helper = {
+	inputIsValid: ( value: string, pattern: string ) => {
+		if ( pattern === null ) {
+			return value !== '' ? Validity.VALID : Validity.INVALID;
+		}
+		return new RegExp( pattern ).test( value ) ? Validity.VALID : Validity.INVALID;
+	},
+	formatPostData: ( form: FormData ): any => {
```

Comment:
We could take the opportunity and use a `Map<string,string>` instead.

Reply:
that seems to break the universe

Reply:
I 💟 breaking universes 💥 ❗️ 

Leave it then, maybe I'll try some time.